core/src/main/java/io/questdb/log/LogAlertSocket.java

```diff
@@ -230,9 +234,9 @@ public boolean send(int len, Consumer<String> ackReceiver) {
                     .$("None of the configured alert managers are accepting alerts.\n")
                     .$("Giving up sending after ")
                     .$(maxSendAttempts)
-                    .$(" attempts: ")
-                    .$(Chars.stringFromUtf8Bytes(outBufferPtr, len))
-                    .$();
+                    .$(" attempts: [")
+                    .$(Chars.stringFromUtf8Bytes(outBufferPtr, outBufferPtr + len))
```

Comment:
could we add a function to the Logger that is able to log utf8 bytes without creating a string?